test/fixtures/time/time.js

```diff
@@ -0,0 +1,55 @@
+const minutesToMilliseconds = [
```

Comment:
Suggestion to please use capital letters and underscores for naming constant variables
Reference - https://javascript.info/variables#uppercase-constants

Reply:
No function in fixtures follows it. I believe it's not really a constant. Is it still suggested?